src/main/resources/feedbackQuestionTextEditFormTemplate.html

```diff
@@ -0,0 +1,10 @@
+<div>
+    <br>
+    <div class="row">
+        <div class="col-sm-4" data-toggle="tooltip" data-placement="top" title="recommended length ">Recommended Length:
+            <input disabled type="number" class="form-control" id="recommendedlength"
+                        name="recommendedlength" value="${recommendedlength}">
+        </div>
+    </div>
+    <br>
```

Comment:
Perhaps this `<br>` could be removed so that the margin from the length input to the bottom of the purple box is equal to the margin from the top of the purple box to the textarea